### DISCUSSION TITLE
Add Options To Make Table Panel look like Logs Panel

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#38152] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I need features to make table panel look alike logs panel.

* disable row and column borders,

* Smaller row-height.

**Why is this needed**:

I open this issue over [this one](https://github.com/grafana/grafana/issues/27530). I think table panel is very important yet very limited compared to other features grafana provides. 

I'm collecting load balancer logs and logs are parsed so that each row separated into different columns. Eg. IP Address, Method, Response Time, so on... This makes it a table and allow me to use the log data on other cases. Yet, when I wanted to display whole log it looks so big, so many padding and margins, font-size also big. It would be great if I can combine tabular data and display it like logs.

**Comment by marefr** [comment]
> Smaller font-size option

Duplicate of #26965 so please remove from this request. 

> It would be great if I can combine tabular data and display it like logs

Use the logs panel. As said in referenced issue, combining multiple features in feature requests makes it hard to understand and reason about. Please remove this from the request.

**Comment by mertbakir** [comment]
> > Smaller font-size option
> 
> Duplicate of #26965 so please remove from this request. 
> 
> > It would be great if I can combine tabular data and display it like logs
> 
> Use the logs panel. As said in referenced issue, combining multiple features in feature requests makes it hard to understand and reason about. Please remove this from the request.

Well I thought all requests as a whole. I don't intend to make multiple requests. Still, removed as you said.

I would use the logs panel if it would work. I'll try to provide a screenshot later.

By combining I meant the existing table panel. It already combines different columns. That's what tables are after all. I'm asking for styling features (eg. font size, padding, row height, borders) to make it look like logs.

Actually, it's about styling options. Logs are just example for my use case. People can customize however they like if we have the options.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
